spec.md

```diff
@@ -1277,6 +1380,7 @@ The following HTTP Headers are defined for this operation:
 | app_guid | string | Deprecated in favor of `bind_resource.app_guid`. GUID of an application associated with the Service Binding to be created. If present, MUST be a non-empty string. |
 | bind_resource | [BindResource](#bind-resource-object) | A JSON object that contains data for Platform resources associated with the Service Binding to be created. See [Bind Resource Object](#bind-resource-object) for more information. |
 | parameters | object | Configuration parameters for the Service Binding. Service Brokers SHOULD ensure that the client has provided valid configuration parameters and values for the operation. |
+| network_data | object | Network configuration data depending on the Network Profile. This object MUST contain a field `network_profile_id`. The value of this field MUST be the ID of the Network Profile. |
```

Comment:
We should separate opaque network data from spec data like network_profile_id.
Could we introduce a `data` key just like in the network_profiles? E.g.
```
...
"network_data": {
  "network_profile_id": "urn:something:vpn",
  "data": {
    "profile-specific-key": "value"
  }
}
...
```